koopovereenkomst-v3-simple/pages/steps_verkoper/Step5.tsx

```diff
@@ -28,119 +23,25 @@ interface Step5Props {
   stepNr: number;
   handleNext: () => void;
   handleBack: () => void;
-  koek: KoopovereenkomstAggregate;
+  koek: KoekAggregate;
 }
 
 export function Step5({ stepNr = 5, handleNext, handleBack, koek }: Step5Props) {
-  const title = "Verkoper Homepage";
 
-  const { session } = useSession();
-  const webId = session.info.webId;
-
-  const [podUrl, setPodUrl] = useState("");
-  const [caseId, setCaseId] = useState("");
-  const [curCase, setCase] = useState<Case>({} as Case);
   const [errors, setErrors] = useState("");
   const [eventLabels, setEventLabels] = useState([]);
-  
+
   useEffect(() => {
     if (koek) {
```

Comment:
Is always `true`? 🤔

Reply:
Je hebt gelijk, dit was gecopy-paste vanaf Step1b. Daar is die check wel nodig, gezien er dan nog niet een koek gekozen is, bij Step5 is dan natuurlijk wel het geval. Dit is dus inderdaad overbodig